### PR TITLE
Add test for PredicateContainer

### LINKED ISSUE #128: Update code coverage for PredicateContainer

### PR DESCRIPTION
**Pull request by jevan13**
## Describe your changes
- Improve code coverage
- Add test for equals and getCombinedPredicates() to check for null Predicates

## Issue ticket number and link
closes #128 

## Checklist before requesting a review
- [X] I have performed a self-review of my code.
- [X] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: src/test/java/seedu/address/model/person/PredicateContainerTest.java [hunk 1]
```diff
@@ -16,51 +17,12 @@
...
         assertTrue(predicateContainer1.equals(predicateContainer1));
 
         // same values -> returns true
-        PredicateContainer predicateContainer3 = new PredicateContainer()
-                .addNameContainsKeywordsPredicate(namePredicate1)
-                .addPhoneContainsKeywordsPredicate(phonePredicate1)
-                .addEmailContainsKeywordsPredicate(emailPredicate1)
-                .addDepartmentContainsKeywordsPredicate(departmentPredicate1)
-                .addRoleContainsKeywordsPredicate(rolePredicate1);
+        PredicateContainer predicateContainer3 = getPredicateContainer1();
```
Comment (LimErnest) [inline src/test/java/seedu/address/model/person/PredicateContainerTest.java:25]: This test might be the same as the one above where you are testing for the same object

File: src/test/java/seedu/address/model/person/PredicateContainerTest.java [hunk 2]
```diff
@@ -147,5 +82,51 @@ public void test_toString() {
         assertEquals(expected, actual);
     }
 
+    public PredicateContainer getPredicateContainer1() {
```
Comment (LimErnest) [inline src/test/java/seedu/address/model/person/PredicateContainerTest.java:89]: Could add comments explaining what getPredicateContainter 1 and 2 do

File: src/test/java/seedu/address/model/person/PredicateContainerTest.java (hunk 1 above)
Comment (jevan13) [reply src/test/java/seedu/address/model/person/PredicateContainerTest.java:25]: The other tests for the same instance (predicateContainer1 against predicateContainer1), whereas this tests for different instances (predicateContainer1 against predicateContainer3), where both predicateContainer contains the same predicates.

File: src/test/java/seedu/address/model/person/PredicateContainerTest.java (hunk 2 above)
Comment (jevan13) [reply src/test/java/seedu/address/model/person/PredicateContainerTest.java:89]: Added!

File: src/test/java/seedu/address/model/person/PredicateContainerTest.java (hunk 1 above)
Comment (LimErnest) [reply src/test/java/seedu/address/model/person/PredicateContainerTest.java:25]: I see now, thanks for the explanation!

**Review by LimErnest — APPROVED** [review]
LGTM!
